api/users/user.service.js

```diff
@@ -2,6 +2,7 @@ const pool = require("../../config/database");
 
 module.exports = {
   getUsers: async (user_sub) => {
+    await pool.query(`SELECT set_config('app.sub','${user_sub}',true)`);
```

Comment:
Where do we enable RLS for the `user` table?

Reply:
i think sir mentioned why i set RLS using 'user_sub'. i got id column as a auto generated column and and use as index value of database manupulations. So i use user_sub as id for data transfers. Since i set RLS to user_sub.